### PR TITLE
docs: command line cli for update commmand

### PR DESCRIPTION
**Pull request by boddumanohar**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: 
- blobber:
- gosdk:
- system_test:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: cmd/update.go [hunk 1]
```diff
@@ -76,12 +76,12 @@ var updateChunkNumber int
 func init() {
 	rootCmd.AddCommand(updateCmd)
 	updateCmd.PersistentFlags().String("allocation", "", "Allocation ID")
-	updateCmd.PersistentFlags().String("remotepath", "", "Remote path to upload")
-	updateCmd.PersistentFlags().String("localpath", "", "Local path of file to upload")
-	updateCmd.PersistentFlags().String("thumbnailpath", "", "Local thumbnail path of file to upload")
-	updateCmd.Flags().Bool("encrypt", false, "(default false) pass this option to encrypt and upload the file")
+	updateCmd.PersistentFlags().String("remotepath", "", "Remote path to update")
```
Comment (hm90121) [inline cmd/update.go]: It should be "Remote path of file to update"

Comment (boddumanohar) [reply cmd/update.go]: done

**Comment by hm90121** [comment]
@boddumanohar LGTM

**Review by dabasov — COMMENTED** [review]
lgtm

**Pull request merged**